src/main/fc/config.c

```diff
@@ -156,7 +156,15 @@ PG_RESET_TEMPLATE(systemConfig_t, systemConfig,
 #endif
 
 #ifdef BEEPER
-PG_REGISTER(beeperConfig_t, beeperConfig, PG_BEEPER_CONFIG, 0);
+PG_REGISTER_WITH_RESET_FN(beeperConfig_t, beeperConfig, PG_BEEPER_CONFIG, 0);
```

Comment:
Well, you can, see, for example, `blackbox.c` for having variable value conditional on a build flag. Also not that the inclusion of a variable in a PG should not be under a build flag, since that will cause compatibility issues if the PG is extended in future.